docs/_includes/api/data-object-create.sh

```diff
@@ -0,0 +1,12 @@
+curl -XPOST https://nuvla.io/api/data-object -H 'content-type:application/json' -b cookies -d '''
+{
+    "template": {
+        "name": "Speed Camera ABC-123",
```

Comment:
No, it works with `name` under `template`
```
$ curl -XPOST https://nuvla.io/api/data-object -H content-type:application/json -b /Users/konstan/code/nuvla/tests/cookies -d '
{
    "template": {
        "name": "Speed Camera ABC-123",
        "description": "Speed Camera ABC-123 Configuration",
        "bucket": "traffic-infra",
        "object": "speed-cameras/ABC-123.config",
        "content-type": "plain/text",
        "credential": "credential/ec203c16-c39e-4949-9980-e52c945eed5f",
        "subtype": "generic"
    }
}'
{
  "status" : 201,
  "message" : "data-object/919d8e93-f015-39de-9771-5590db29350b created",
  "resource-id" : "data-object/919d8e93-f015-39de-9771-5590db29350b"
}

$ curl https://nuvla.io/api/data-object/919d8e93-f015-39de-9771-5590db29350b -b /Users/konstan/code/nuvla/tests/cookies | jq .
{
  "description": "Speed Camera ABC-123 Configuration",
  "updated": "2022-01-10T13:31:45.643Z",
  "name": "Speed Camera ABC-123",
  "credential": "credential/ec203c16-c39e-4949-9980-e52c945eed5f",
...
```

If this is not expected behavior - please open an issue on the api-server.

Reply:
Checked it works because the data-object generic template doesn't have a name, neither a description. 
What happens with templates is that we resolve href of the template and the content of the template get merged into the the request body. All templates have one except data-objects templates. 

To avoid confusion and be more regular with existing resources, you should put `name` and `description` outside the template.

Reply:
See 1282eb4